doc/src/manual/multi-threading.md

```diff
@@ -1,9 +1,6 @@
-# [Multi-Threading (Experimental)](@id man-multithreading)
+# [Multi-Threading](@id man-multithreading)
```

Comment:
Does this mean we are taking off the Experimental label for 1.5? Should `threadcall` still have the Experimental label on it (like it used to)?

Reply:
It's not SO experimental that it needs to be in the title of the chapter :)